pepys_import/database/default_data/Help Texts.csv

```diff
@@ -0,0 +1,20 @@
+id,guidance
+1,RESOLVE_DATAFILE_TYPE
+2,RESOLVE_PRIVACY
+3,RESOLVE_DATAFILE
+4,RESOLVE_PLATFORM
```

Comment:
@BarisSari - this isn't quite the structure I was expecting.  I was hoping for a legible ID.

```
id, guidance
RESOLVE_NATIONALITY,  The nationality of the platform.  Note: while a platform may have a fictional nation created  for exercise purposes - this should be the true nation it belongs to.  The fictional role undertaken by it should be captured within the Participation data.
RESOLVE_SENSOR_TYPE, The category of sensor.  These categories are steadily maturing, but please see an expert user if the relevant category isn't present.
```

Reply:
When you said ID, I thought you meant Integer type. Yes, I think this structure is more sensible. I'll make changes for that.

By the way, I didn't write the guidance texts, yet. Please don't pay attention to that.